tests/data/metadata/fence_ibm_powervs.xml

```diff
@@ -75,6 +83,7 @@
 	</parameter>
 	<parameter name="debug_file" unique="0" required="0" obsoletes="debug">
 		<getopt mixed="-D, --debug-file=[debugfile]" />
+		<content type="string"  />
```

Comment:
You need to remove this (`make xml-upload` should solve it).

It's a special case to avoid detecting default path for files (which might change based on values from ./configure).

Reply:
Removed content type="string" from debug_file parameter.